src/commands/registries/azure/tasks/scheduleRunRequest.ts

```diff
@@ -24,7 +24,12 @@ import { getStorageBlob } from '../../../../utils/lazyPackages';
 const idPrecision = 6;
 const vcsIgnoreList = ['.git', '.gitignore', '.bzr', 'bzrignore', '.hg', '.hgignore', '.svn'];
 
-export async function scheduleRunRequest(context: IActionContext, requestType: 'DockerBuildRequest' | 'FileTaskRunRequest', uri: vscode.Uri | undefined): Promise<() => Promise<AcrRun>> {
+export enum RootStrategy {
```

Comment:
It's repeated in the code below but can we also add a comment here to note that this enum is shared with the AKS DevX extension?